src/css/custom.css

```diff
@@ -22,6 +22,18 @@ body {
         rgba(0, 0, 0, 0.117647) 0px 1px 4px;
 }
 
+.search-bar {
+  margin-bottom: 24px;
+  width:100% !important;
+}
+
+/* images */
+.icon-search {
+  position: absolute;
```

Comment:
The work around for achieving what you are doing here that we did in Mobile was to wrap the icon and the Text field in a div (View in RN) and put a bottom border on that. Not sure if that is exactly better, but absolute positioning is a source of nightmares when you have dynamic sizes in RN at least.

It's a shame that Material-ui doesn't have a search field or generic text field component that comes with an icon :/